docs/concepts/personal-access-token.mdx

```diff
@@ -62,6 +62,76 @@ Accept: application/json
 Authorization: Bearer $API_KEY
 ```
 
+## Authorizing Ory Network Workspace Admin APIs
+
+Workspace API keys are used to authorize privileged operations to Ory Network Workspace admin APIs as well as the project admin
+APIs of all projects that belong to this workspace. For example, you must use a Workspace API key to create a new project in a
+workspace, or add a new B2B SSO organization to a project within that workspace.
+
+API Keys work in the context of the workspace for which they were created, as well as for all projects of that workspace. You
+create API Keys in the [Ory Console](https://console.ory.sh/).
+
+However, workspace API keys do not work for managing the data of the projects itself, such as interacting with identities,
+sessions, or OAuth2 clients. For these operations, you need to use the
+[project API keys](#authorizing-ory-network-project-admin-apis).
+
+### Creating API keys
+
+Follow these steps to create API keys for your workspace:
+
+[//]: # "TODO: use the <ConsoleLink> once it works with `current` workspace links"
+
+1. Go to the [Ory Console](https://console.ory.sh/).
+2. Go to **Workspace settings** through the workspace menu in the top left corner.
+3. Select **API keys** in the sidebar.2. Click the **+** icon in the **API Keys** section.
+4. Enter the API Key name and click **Create new API key**.
+5. Copy the created API Key from the prompt that shows at the bottom right of the screen.
+
+:::caution
+
+The API Key you created is displayed only once. When you close the prompt, you can't access the key again. Make sure to save the
+API Key before you close the prompt or reload the page.
+
+API Keys are bound to an Ory Network project, and not to a specific user. This means that any user with access to the API Key can
+perform the operations that the API Key allows, even if they are no longer part of the project. Make sure to keep your API Keys
+secure and rotate them if necessary.
+
+:::
+
+### Usage
+
+Use the API Key in API calls, SDK calls, or command-line interactions. Ory Workspace API Keys have a `ory_wak_` prefix, which
+makes it easy to identify them when analyzing code.
+
+For example, when calling the API to get a project at `/projects/:project_id`, include the API Key in the `Authorization` header:
+
+```shell
+GET /projects/$PROJECT_ID HTTP/1.1
+Host: api.console.ory.sh
+Accept: application/json
+# highlight-next-line
+Authorization: Bearer $API_KEY
+```
+
+## Workspace API Keys vs Project API Keys
+
+Workspace API keys are used to authenticated privileged operations on the workspace level, such as creating a new project in a
+workspace or adding a new B2B SSO organization to a project within that workspace. Project API keys are used to authenticate
+privileged operations on the project level, such as listing identities or creating OAuth2 clients.
```

Comment:
it's a copy paste of description from above, I think we can remove it as you have it already in  ```## Authorizing Ory Network Workspace Admin APIs```